COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -925,7 +805,7 @@ There are two recommended methods for installing the Zowe CLI. If you have acces
 Issue the following command in your terminal (e.g. Command Prompt or if you are using VS Code, Terminal -> New Terminal):
 
 ```
-npm install -g @zowe/cli@zowe-v1-lts
+npm install -g @zowe/cli@zowe -v3-lts
```

Comment:
There's a space between `zowe` and `-v3`

Reply:
Fixed issue!